recipe/meta.yaml

```diff
@@ -94,25 +91,27 @@ outputs:
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
         - cuda-version {{ cuda_version }}
         - cuda-nvcc-tools {{ cuda_version }}
+        - {{ pin_subpackage("cuda-crt-dev_" + target_platform, exact=True) }}
+        - {{ pin_subpackage("cuda-nvvm-dev_" + target_platform, exact=True) }}
```

Comment:
Same question as above re: why this is needed.

Reply:
Let's try dropping these and see what happens

```suggestion
```

Reply:
As these are `-dev` packages, they would make sense to keep (especially if there is something linking to them)